File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
#include <algorithm>

using namespace std;

bool solution(string s) {
    int pcount = count_if(s.begin(), s.end(), [](char c){ return tolower(c) == 'p'; });
    int ycount = count_if(s.begin(), s.end(), [](char c){ return tolower(c) == 'y'; });
    
    return pcount == ycount;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    ycount = 0
    pcount = 0
    for i in s:
        if i == 'p' or i == 'P':
            pcount += 1
        elif i == 'y' or i == 'Y':
            ycount += 1
            
    return ycount == pcount